index.html

```diff
@@ -0,0 +1,146 @@
+            <div class="card-program">
+              <i class="fa-solid fa-camera-retro program-icon"></i>
+              <h3 class="program-item">Meet & Greet</h3>
+              <p class="program-desc">Meet, share, take photos, ask for his autograph and tell him what makes you feel.</p>
+            </div>
+
+          </div>
+            <a href="#" class="whole">SEE THE WHOLE CONCERT</a>          
+            <a class="join-program" href="#">Join to MCM World Tour 2023</a>
+          </div>
+      </section>
+      <section class="featured-singer">
+        <div class="signers-container">
+          <h2 class="singer-tittle">Featured Singers</h2>          
+          <div class="singers-cards">
+          </div>
+          <div class="btn-cont to-show">
+            <button class="btn-more">MORE <span></i><i class="fa-solid fa-chevron-down"></i></span></button>
+          </div>
+        </div>        
+      </section>      
+    </main>
+    <footer>
+      <div class="partner">
+        <h2>Partner</h2>
+        <hr class="small-line">
+        <div class="logos">
+          <a href="https://www.mozilla.org/es-ES/firefox/new/"><img class="logo-partner" src="images/log-moz.png" alt="Mozilla Logo"></a>
+          <a href="https://www.google.com/"><img class="logo-partner" src="images/log-google.png" alt="Google Logo"></a>
+          <a href="https://www.naver.com/"><img class="logo-partner" src="images/log-naver.png" alt="Naver Logo"></a>
+          <a href="https://www.kakaocorp.com/page/"><img class="logo-partner" src="images/log-daum.png" alt="Daumkakao Logo"></a>
+          <a href="https://www.airbnb.com/"><img class="logo-partner" src="images/log-airbnb.png" alt="Airbnb Logo"></a>          
+        </div>
+      </div>
+      <div class="copyright">
+        <div class="cop-img">
+          <img class="synod" src="images/logo-sinodo.png" alt="Logo">
+        </div>        
+        <div class="cop-desc">
+          <p class="cop-text">2023 - Synod 21-24 - Concert Music Catholic</p>
+          <p class="cop-text">We recall that the purpose of the Synod is not to produce documents, but to plant dreams, draw forth prophecies and visions</p>
+        </div>
+      </div>
+    </footer>    
+  </div>
+  <!-- menu modal   -->
+  <div id="container-menu-modal">
+    <div id="modal">
+        <div id="menu-head">
+          <i class="fa-solid fa-xmark" id="btn-close"></i>
+        </div>
+        <div>
+            <ul id="modal-ul">
+                <li class="modal-item"><a href="about.html">About</a></li>
+                <li class="modal-item"><a href="#Program">Program</a></li>
+                <li class="modal-item"><a href="#Sponsor">Sponsor</a></li>
+                <li class="modal-item"><a href="#News">News</a></li>                
+            </ul>                
+        </div>
+        <hr class="endline">
+    </div>
+</div>
+<script src="js/menu.js"></script>
+<script src="js/singers.js"></script>
+</body>
+</html>
```

Comment:
- your footer should be aligned to the centre and padding should be present between the items in the footer. 👍🏾 

- as suggested your video presentation previously should be less than 5mins currently it's over 5mins(10mins) please rerecord your video and put it in your readme file.

- you should add a hover effect on your main program section to improve the UI of your webpage with a white border on each card.

![image](https://user-images.githubusercontent.com/60612329/227641359-73ff5445-708e-4fd2-8f64-2ad8d28847ad.png)

Reply:
Hi @alexander16108, I am thinking that this is a type of discrimination, I made the changes that were presented and now there are others, but this had never happened. These changes you suggest are not part of the requirements.

Reply:
My video is in the readme file and the duration is 5 minutes
https://www.loom.com/share/8c1d08ddbe584779b80f1a2132df8e56